README.md

```diff
@@ -40,6 +40,11 @@ yarn dev
 ```bash
 yarn lint
 ```
+### Run tests
+
+```bash
+npx jest
```

Comment:
Should be `yarn test` for consistency and to avoid additional installation of npx

Reply:
Ty